clone_all.sh

```diff
@@ -64,7 +64,7 @@ EXTENSIONS=(
   "WikibaseExport master https://github.com/ProfessionalWiki/WikibaseExport.git"
   "ExternalContent master https://github.com/ProfessionalWiki/ExternalContent.git"
   "SPARQL master https://github.com/ProfessionalWiki/SPARQL.git"
-  "WikibaseFacetedSearch master https://github.com/ProfessionalWiki/WikibaseFacetedSearch.git"
+  "WikibaseFacetedSearch master https://github.com/MaRDI4NFDI/WikibaseFacetedSearch.git"
```

Comment:
I noticed this does not match the PR description

Reply:
Thanks, that was fixed in a follow-up.